docs/README.rst

```diff
@@ -16,3 +16,6 @@ browser.
 
 .. _ReST: https://docutils.sourceforge.io/rst.html
 .. _Sphinx: https://www.sphinx-doc.org/
+
+.. [1] It's necessary to run this command from the project top level folder because, due to the viewcode extension, the
+   docs are dependent on Django itself and Django is installed by reference from your local directory.
```

Comment:
I pushed edits, does it read better now?

Reply:
I had assumed we needed the Django installed to exactly match the current codebase, but this doesn't appear to be the case (I believe the driver of needing Django installed to have apps so that we can import files when the viewcode extension is trying to import each file).

I have now simplified how Django is installed in the `requirements.txt` file and all of the documentation on how to install. 
Thank you for all your reviews so far, I'm definitely learning from it :star: